src/Viewer/renderers/point.jsx

```diff
@@ -439,6 +439,16 @@ export default React.memo(function Point(props) {
 
 })
 
+function normalizeSize(size, style) {
+  if (style === "triangle") {
+    return size * 1.5;
+  } else if (style === "diamond") {
+    return size * 1.5;
+  } else if (style === "square" || style === "circle") {
+    return 5;
```

Comment:
Don't hard code the size to 5 or we can't change the size with a `<styleDefinition>` tag!

Reply:
When I return `size` (which is supposed to be 5 for `square` and `circle`) instead of 5, the sizes of `square` and `circle` are incorrect. I haven't been able to figure out why...
![Screen Shot 2022-11-04 at 11 38 44 AM](https://user-images.githubusercontent.com/72275793/199964123-632253b2-7054-4953-9f22-f186aeb61b32.png)
<img width="948" alt="Screen Shot 2022-11-04 at 11 30 52 AM" src="https://user-images.githubusercontent.com/72275793/199964136-f6ff0c75-0ba3-450b-84a6-d43063a2134a.png">

Reply:
Something strange going on.  It works fine for me when I check out your branch.  Make sure you build core after changing style.js.  Maybe clear cache.